### ISSUE TITLE
/engine endpoint should accept uploads to any container level

### ISSUE THREAD

**Issue by gsfr** [scitran/core#160]
We currently only accept uploads to acquisitions.

We desire the following functionality:
- Uploads to an acquisition, its parent session and parent project
- Uploads to a session and its parent project
- Uploads to a project (not sure this is actually a thing)

**Comment by kofalt** [comment]
Tentatively blocked by #159 (not a hard requirement, but might make life easier).

**Comment by kofalt** [comment]
@nagem you were asking about this limitation earlier? Here's the ticket tracking that.

**Comment by nagem** [comment]
As of #357, you are able to upload to an acquisition, a session, or a project. Uploading to multiple levels of the hierarchy at the same time, if still necessary, can be done in a separate PR.

**Comment by kofalt** [comment]
SGTM. We don't have anything on the roadmap regarding gears simultaneously uploading to multiple levels of the hierarchy.

**Comment by gsfr** [comment]
We should have our schemas enforce this limitation.
